src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
       });
   }
 
+  handleSelectChange = selectedOption => {
+    this.alertsSelectionArray = selectedOption;
+    this.setState({
+      alertSignUps: this.alertsSelectionArray
+        .map(item => item.full_fips_code)
+        .join(','),
+    });
+  };
+
   render() {
     const { stateId, county } = this.props;
+
+    const stateDataset = US_STATE_DATASET.state_dataset.map(state => {
+      return {
+        ...state,
+        full_fips_code: state.state_fips_code + '000',
+      };
+    });
+    const countyDataset = [];
+
+    each(US_STATE_DATASET.state_county_map_dataset, (value, key) => {
+      countyDataset.push(...value.county_dataset);
+    });
+
+    const autocomplete_options = stateDataset.concat(countyDataset);
+
     return (
       <StyledNewsletter>
         <form
           ref={f => (this.form = f)}
           className="js-cm-form"
           id="subForm"
           method="post"
-          data-id="2BE4EF332AA2E32596E38B640E90561930C9C3A433D015D9D4BD88E99175E51395EF5EBFFD527179E032AC15455BB1208D87A6CE87843E524B0EA520CBFF446E"
+          data-id="2BE4EF332AA2E32596E38B640E905619E90CD5DAC48A878CDEBFFE3B420D8CD24E4AEABAB52A4CE3526219C7A966AE965B84F99C823C89EFF1F01B28DE4F975E"
         >
           <input
             hidden
             readOnly
             aria-label="state"
             value={stateId || ''}
-            id="fieldjlkiyul"
-            maxLength="200"
-            name="cm-f-jlkiyul"
+            id="fieldjrdtwi"
+            maxlength="200"
+            name="cm-f-jrdtwi"
           />
           <input
             hidden
             readOnly
             aria-label="county"
+            id="fieldjrdtwd"
+            maxlength="200"
+            name="cm-f-jrdtwd"
             value={county || ''}
-            id="fieldjlkulhk"
-            maxLength="200"
-            name="cm-f-jlkulhk"
           />
+          <InputHolder>
+            <input
+              type="checkbox"
+              value="wurhhh"
+              id="wurhhh"
+              name="cm-ol-wurhhh"
+            />
+            <label for="checkbox">
+              {' '}
+              Also send me <b>daily news</b> with the latest data and scientific
+              findings on COVID{' '}
+            </label>
+          </InputHolder>
           <input
-            ref={i => (this.emailInput = i)}
-            autoComplete="Email"
-            aria-label="Email"
-            placeholder="Enter your email address"
-            className="js-cm-email-input qa-input-email"
-            id="fieldEmail"
-            maxLength="200"
-            name="cm-wurhhh-wurhhh"
-            required=""
-            type="email"
+            hidden
+            aria-label="alert_list_csv"
+            id="fieldjrdtwy"
+            maxlength="200"
+            name="cm-f-jrdtwy"
+            onChange={value => {
+              console.log(value);
+            }}
+            value={this.state.alertSignUps}
+          />
+          <Autocomplete
```

Comment:
Nit: I'd like to be able to just type "Washi<enter>" to add Washington, but I have to actually click on it or press down arrow and then enter or whatever...  Maybe if we set `autoHighlight` and/or `autoSelect` it'll work?

Reply:
Maybe kinda' Related: Right now, pressing <enter> actually seems to submit the form which maybe we don't want?

Reply:
hmmmm i'm not seeing that >.<

Reply:
Blah, sorry. I didn't escape my &lt; and &gt; signs so my message didn't make sense.  😞 

What I meant is that I should be able to type "Wash&lt;enter&gt;" to add "Washington" but instead I get sent to an error page that says "Invalid email address" because it submits the form.